File: platform/sdk/nodejs/examples/cloud/cloud-016_iam_put_policy.js
const { CloudClient } = require("../../mox-sdk-cloud");
const client = new CloudClient({ region: "cn-east-1" });
const doc = { Version: "2012", Statement: [{ Effect: "Allow", Action: "s3:*" }] };
const result = client.iamPutPolicy("AdminPolicy", doc);
if (!result.ok || !result.version) process.exit(1);
console.log("XJ-OK: cloud-016_iam_put_policy");
process.exit(0);

File: platform/sdk/python/examples/cloud/cloud-016_iam_put_policy.py
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from mox_sdk_cloud import CloudClient

client = CloudClient({"region": "cn-east-1"})
doc = {"Version": "2012", "Statement": [{"Effect": "Allow", "Action": "s3:*"}]}
result = client.iamPutPolicy("AdminPolicy", doc)
if not result["ok"] or not result["version"]:
    sys.exit(1)
print("XJ-OK: cloud-016_iam_put_policy")
sys.exit(0)
